pkgs/tools/networking/ht-rs/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, rustPlatform, fetchFromGitHub, Security }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ht-rs";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "ducaale";
+    repo = "ht";
+    rev = "v${version}";
+    sha256 = "jT4Y6Yr3Cs2YPDSPNxTdIbyl6dP8v+HqG2zQJnPHvHA=";
+  };
+
+  cargoSha256 = "sha256-vx619B1Bv9yiGeL338PzzNyXAL0uj6eWzaaR3g4zvFo=";
+
+  buildInputs = [ ] ++ lib.optional stdenv.isDarwin Security;
+
+  postInstall = ''
+    mv $out/bin/ht $out/bin/ht-rs
```

Comment:
Should we really also rename the binary? Is upstream or any downstream distribution doing the same? How likely is it to have boot binaries in the same package? The issue with that is that it might break compatibility issues in case some other package depends on the binary named `ht`.

Reply:
We have another package and its binary named ht. I wanted to avoid similar scenarios happening with old one, hence rename. If you suggest to keep as is, I'm fine either way.

Reply:
The other package was never updated since added to nixpkgs in 2015. I don't think it has a lot of users and hence naming it ht. If you disagree with me, you could still create a symlink from ht to ht-rs. This would allow to access both executables if they end up in the same profile.

Reply:
Also I just realized the other package is broken. I fill a PR to get it removed.

Reply:
Added symlink and removed rename.